Add ability to change UI background color (#121)

fabiocolacio opened the pull request:
The default color is set to black. This might not be desirable but you can change the gschema to a better color later.

I also added the settings in the preferences to set this.

![custom-background2](https://user-images.githubusercontent.com/5248352/77838366-1dc06500-7128-11ea-92fb-9dbc21264f37.png)

maoschanz commented:
>The default color is set to black. This might not be desirable

that's why i told you to set it to transparent by default :(

Anyway is it really required to do this allocation thing? I would prefer to just `paint()`

fabiocolacio commented:
I changed the default color to be completely transparent so it shows the normal widget color like before.

I also removed the rectangle code as you suggested. I thought it was necessary but it turns out I was wrong :)

maoschanz commented:
>These conflicts are too complex to resolve in the web editor

>Use the command line to resolve conflicts before continuing.

...and the command line didn't even detect a single conflict. Just github being github.
